documentation/meta/traffic/runbooks/identifying-and-blocking-traffic-anomalies.md

```diff
@@ -0,0 +1,163 @@
+```
+
+````{warning}
+If you haven't found anything, it's unlikely that this situation is connected to malicious traffic or specific actors. Instead, it is more likely to be a normal increase in traffic to Openverse. At this point, it's more likely that a change in code or infrastructure has caused a performance drop, exposed by the higher traffic, or that the traffic has grown so much that we need to allocate additional server resources. Consider recent Openverse marketing efforts or WordPress events if the traffic looks organic and we need to attribute it to something.
+
+2.  Take the list of suspicious traffic source IDs and go to the
+    `Security => Bots` section of Cloudflare. Here, you'll see more lists of IP
+    Addresses, ASNs, and User Agents below a graph of known bot traffic. It's
+    very likely you'll see items from your lists in these graphs.
+
+3.  Use the "add filter" button and filter by your listed resources
+    individually. Write down which of the identified resources are comprised of
+    the most `Automated` and `Likely Automated` traffic.
+
+```{warning}
+If an ASN has a significant amount of human traffic, it can indicate this ASN is an ISP rather than a hosting company or company which offers a Virtual Private Network. Blocking these types of ASNs can disrupt service to human users.
+````
+
+### 3. Block suspicious traffic sources in the Web Access Firewall
+
+For each source in your list, create a Web Access Firewall rule in Cloudflare.
+These can be found in the `Security => WAF` section. The
+[official Cloudflare docs](https://developers.cloudflare.com/firewall/cf-dashboard/create-edit-delete-rules/#create-a-firewall-rule)
+explain how to do this well. Some suggestions:
+
+- Add the type of source being blocked to the name of the rule in brackets. For
+  example, we currently use a `[bot-asn]` name prefix for all rules relating to
+  ASNs.
+- When writing rules which match ASNs, only the 5-digit number of the ASN is
+  used in the rule. For this reason, it's nice to add the name of the ASN to the
+  rule name as well. For example, rule name `[bot-asn] ZackWorld` which blocks
+  ASN `13335`.
+- Default to the "Block" action for each firewall rule.
+- If a source id had a high percentage of human traffic and is making requests
+  to the frontend, but you still feel it should be blocked, consider using the
+  "Managed Challenge" action instead of an outright block. This should allow
+  human users but block automated traffic.
+
+### 4. Analyze reputation of identified traffic sources
+
+Finally, make a web search in your preferred browser for each individual traffic
+source identifier. Many ASNs and IP addresses used for scraping will have
+negative reports, be on spam lists, or be associated with hosted VPN services.
+This can be helpful in identifying which are truly malicious.
```

Comment:
Should this perhaps be step 3 instead? So that it comes before the instructions for blocking a traffic source.

Reply:
It actually was previously, but I moved it because it can be time consuming and I don't want it to block mitigating the issue. It's cool that this stood out to you though; I will add information explaining its positioning!

Reply:
Ah okay, sounds good!